### DISCUSSION TITLE
Welcome to kutt Discussions!

### DISCUSSION THREAD

**Discussion by poeti8** [thedevs-network/kutt#414] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maud-lv** [comment]
Hello, I'm Maud, I'm learning to code. First of all I wanted to say kudos for kutt.it. I discovered it recently and I love it! I just finished a course on SQL and Python. Later on I plan on learning Javascript. I speak English and French. If you can see any area where I could help contribute somehow let me know, I would be glad to do so.
